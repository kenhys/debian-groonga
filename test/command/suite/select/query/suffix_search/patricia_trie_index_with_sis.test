table_create People TABLE_NO_KEY
column_create People name COLUMN_SCALAR ShortText
table_create Names TABLE_PAT_KEY|KEY_WITH_SIS ShortText
column_create Names people_index COLUMN_INDEX People name

load --table People
[
{"name": "ひろゆき"},
{"name": "まろゆき"},
{"name": "ひろあき"},
{"name": "ゆきひろ"}
]
select --table People --query name:$ゆき
