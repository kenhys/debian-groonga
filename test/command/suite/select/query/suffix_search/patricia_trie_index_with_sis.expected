table_create People TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create People name COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Names TABLE_PAT_KEY|KEY_WITH_SIS ShortText
[[0,0.0,0.0],true]
column_create Names people_index COLUMN_INDEX People name
[[0,0.0,0.0],true]
load --table People
[
{"name": "ひろゆき"},
{"name": "まろゆき"},
{"name": "ひろあき"},
{"name": "ゆきひろ"}
]
[[0,0.0,0.0],4]
select --table People --query name:$ゆき
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        2
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "name",
          "ShortText"
        ]
      ],
      [
        2,
        "まろゆき"
      ],
      [
        1,
        "ひろゆき"
      ]
    ]
  ]
]
